_posts/2021-06-01-release-1.13.1.md

```diff
@@ -0,0 +1,162 @@
+---
+layout: post
+title:  "Apache Flink 1.13.1 Released"
+date:   2021-06-01 00:00:00
+categories: news
+authors:
+- dwysakowicz:
+  name: "Dawid Wysakowicz"
+  twitter: "dwysakowicz"
+
+---
+
+The Apache Flink community released the first bugfix version of the Apache Flink 1.13 series.
+
+This release includes 82 fixes and minor improvements for Flink 1.13.1. The list below includes a detailed list of all fixes and improvements.
```

Comment:
When I search this page for `browse/FLINK-`, I have 52 hits, not 82.

```suggestion
This release includes 52 fixes and minor improvements for Flink 1.13.1. The list below includes a detailed list of all fixes and improvements.
```

Reply:
According to some internal discussions, I removed all documentation and test instabilities from the list below. The full release notes can be accessed in JIRA. I probably should've changed the wording here as well. WDYT about:
> The list below includes most important changes. For full list see: https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12315522&version=12350058